person_class.rb

```diff
@@ -5,7 +5,7 @@
 class Person < Nameable
   # Getters&setters for @id, @name, and @age
   attr_reader :id
-  attr_accessor :name, :age
+  attr_accessor :name, :age, :rentals
 
```

Comment:
- [ ] Kindly make sure to add the many-to-many relationship method between the `Person` and `Rentals` class, to the `Person` class. You can refer to the [following](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship) example.